lib/v2/tesla/price/index.js

```diff
@@ -0,0 +1,59 @@
+const cheerio = require('cheerio');
+const got = require('got');
+
+module.exports = async (ctx) => {
+    const model3_link = 'https://www.dongchedi.com/auto/series/3762';
+    const modely_link = 'https://www.dongchedi.com/auto/series/4363';
+
+    const model3_response = await got({
+        method: 'get',
+        url: model3_link,
+    });
+
+    const modely_response = await got({
+        method: 'get',
+        url: modely_link,
+    });
+
+    const $model3 = cheerio.load(model3_response.body);
+    const $modely = cheerio.load(modely_response.body);
+
+    let model3_price = '';
+    let modely_price = '';
+
+    for (let i = 0; i < $model3('p').length; i++) {
+        const $ele = $model3('p').eq(i);
+        if ($ele.text().trim().indexOf('经销商报价') > -1) {
+            const priceDom = $ele.next();
+            model3_price = ' => 经销商报价: ' + $model3(priceDom).text().trim();
+            break;
+        }
+    }
+
+    for (let i = 0; i < $modely('p').length; i++) {
+        const $ele = $modely('p').eq(i);
+        if ($ele.text().trim().indexOf('经销商报价') > -1) {
+            const priceDom = $ele.next();
+            modely_price = ' => 经销商报价: ' + $modely(priceDom).text().trim();
+            break;
+        }
+    }
```

Comment:
Is it possible to extract a common function? So that when adding more tesla models later, there won't be repeated code.

Reply:
ok

Reply:
can merge this pull request now